testing/python/test_closures.py

```diff
@@ -0,0 +1,145 @@
+"""WIP: this test class will be moved to fixture.py in this module"""
+
+import pytest
+
+from _pytest.fixtures import FixtureRequest
+
+
+class TestScopeOrdering(object):
+
+    @pytest.mark.parametrize('use_mark', [True, False])
+    def test_func_closure_module_auto(self, testdir, use_mark):
```

Comment:
The only test in this module which *passes* on `master` is this one when `use_mark=False`: in this case, both `m1` and `f1` are `autouse`, which we **do** sort by scope on `master`:

https://github.com/pytest-dev/pytest/blob/1b535387bfc68bd8d0e1971558d5d7ac9f9e129a/_pytest/fixtures.py#L1017-L1019